test/parse.jl

```diff
@@ -246,7 +246,8 @@ for T in vcat(subtypes(Signed), subtypes(Unsigned))
 
     # Test that the entire input string appears in error messages
     let s = "     false    true     "
-        result = @test_throws ArgumentError get(Base.tryparse_internal(Bool, s, start(s), endof(s), 0, true))
+        result = @test_throws(ArgumentError,
```

Comment:
I think I'd leave this one as it was. It's no longer than base/sysimg.jl line 145, for example, and is a little easier to follow when it's on one line (IMO).